nifi-nar-bundles/nifi-poi-bundle/nifi-poi-processors/src/main/java/org/apache/nifi/processors/poi/ConvertExcelToCSVProcessor.java

```diff
@@ -313,7 +334,7 @@ private String getSheetsNotFound(Map<String, Boolean> desiredSheets) {
      * do most of the work of parsing the contents of the Excel sheet
      * and outputs the contents as a (basic) CSV.
      */
-    private static class SheetToCSV {
+    private class SheetToCSV {
```

Comment:
@exceptionfactory Shouldn't this class be static?

Reply:
That was a good change, but there was an `e.printStackTrace()` in a catch block that I changed to call `getLogger().warn()`, requiring the class to be non-static. The other option is to pass the ComponentLog reference to the SheetToCSV class, but since the class was previously non-static, this seemed like a less impactful change. What do you think?

Reply:
@exceptionfactory I did not realize that. The way you have it then is fine. My motto keep it simple :)

Reply:
Thanks @dan-s1, I appreciate the attention to detail!